index.js

```diff
@@ -30,14 +30,15 @@ module.exports = class Localdrive {
     }
 
     if (stat.isSymbolicLink()) {
-      entry.value.linkname = (await fsp.readlink(filename)).slice(this.root.length)
+      const link = (await fsp.readlink(filename)).slice(this.root.length)
+      entry.value.linkname = unixPathResolve('/', link)
```

Comment:
ah, we should support relative paths here

Reply:
ie replace the \ with /, that’s it

Reply:
Should I still remove the root from the `linkname`?
About resolving the link, it's because I treated `linkname` as any other key